test/server/graph/mutations/removeCommentTag.js

```diff
@@ -34,34 +34,53 @@ describe('graph.mutations.removeCommentTag', () => {
     const context = new Context({user});
 
     // add a tag first
-    await CommentsService.addTag(comment.id, 'BEST');
+    await CommentsService.addTag(comment.id, 'BEST', user.id);
     const response = await graphql(schema, query, {}, context, {id: comment.id, tag: 'BEST'});
     if (response.errors && response.errors.length) {
       console.error(response.errors);
     }
     expect(response.errors).to.be.empty;
     expect(response.data.removeCommentTag.errors).to.be.null;
-    expect(response.data.removeCommentTag.comment.tags).to.deep.equal([]);
+
+    CommentsService.findById(response.data.removeCommentTag.comment.id)
```

Comment:
Should use `await` here, it escapes context by not calling return/await and we won't capture the error.